includes/admin/payments/orders.php

```diff
@@ -881,42 +996,55 @@ function edd_order_details_attributes( $order ) {
-					<input type="text" id="edd-payment-date" name="edd-payment-date" value="<?php echo esc_attr( $order_date->format( 'Y-m-d' ) ); ?>" class="medium-text edd_datepicker" placeholder="<?php echo esc_attr( edd_get_date_picker_format() ); ?>"/>
+					<input type="text" id="edd-payment-date" name="edd-payment-date" value="<?php echo esc_attr( $order_date->format( 'Y-m-d' ) ); ?>" class="edd_datepicker" placeholder="<?php echo esc_attr( edd_get_date_picker_format() ); ?>"/>
 
 				</div>
 
 				<div class="edd-admin-box-inside">
-					<span class="label">
-						<?php echo esc_html( __( 'Time', 'easy-digital-downloads' ) . ' (' . edd_get_timezone_abbr() . ')' ); ?>
-					</span>
-
-					<label for="edd_payment_time_hour" class="screen-reader-text">Hour</label>
-					<?php
-					echo EDD()->html->select( array(
-						'name'             => 'edd-payment-time-hour',
-						'id'               => 'edd-payment-time-hour',
-						'options'          => edd_get_hour_values(),
-						'selected'         => $order_date->format( 'H' ),
-						'chosen'           => true,
-						'class'            => 'edd-time',
-						'show_option_none' => false,
-						'show_option_all'  => false,
-					) ); // WPCS: XSS ok.
-					?>
-					:
-					<label for="edd_payment_time_min" class="screen-reader-text">Minute</label>
-					<?php
-					echo EDD()->html->select( array(
-						'name'             => 'edd-payment-time-min',
-						'id'               => 'edd-payment-time-min',
-						'options'          => edd_get_minute_values(),
-						'selected'         => $order_date->format( 'i' ),
-						'chosen'           => true,
-						'class'            => 'edd-time',
-						'show_option_none' => false,
-						'show_option_all'  => false,
-					) ); // WPCS: XSS ok.
-					?>
+					<fieldset class="edd-form-group">
+						<legend class="edd-form-group__label">
+							<?php echo esc_html( __( 'Time', 'easy-digital-downloads' ) . ' (' . edd_get_timezone_abbr() . ')' ); ?>
+						</legend>
+
+						<div class="edd-form-group__control">
+							<label for="edd-payment-time-hour" class="screen-reader-text">
+								<?php esc_html_e( 'Hour', 'easy-digital-downloads' ); ?>
+							</label>
+
+							<?php
+							echo EDD()->html->select(
+								array(
+									'name'             => 'edd-payment-time-hour',
+									'id'               => 'edd-payment-time-hour',
+									'options'          => edd_get_hour_values(),
+									'selected'         => $order_date->format( 'H' ),
+									'chosen'           => true,
+									'class'            => 'edd-time edd-form-group__input',
+									'show_option_none' => false,
+									'show_option_all'  => false,
+								)
+							); // WPCS: XSS ok.
+							?>
+							:
+
+							<label for="edd-payment-time-min" class="screen-reader-text">
+								<?php esc_html_e( 'Minute', 'easy-digital-downloads' ); ?>
+							</label>
+							<?php
+							echo EDD()->html->select(
+								array(
+									'name'             => 'edd-payment-time-min',
```

Comment:
An `id` parameter needs to be added here--the `select` method will not automatically populate it.